core/src/main/java/org/frankframework/http/mime/MessageContentBody.java

```diff
@@ -70,27 +70,15 @@ public String getFilename() {
 
 	@Override
 	public void writeTo(OutputStream out) throws IOException {
-		int length = Math.toIntExact(getContentLength());
+		long length = getContentLength();
```

Comment:
Dit lijkt wel heel erg op wat je hierboven in `HttpMessageEntity#writeTo` hebt aangepast. Kan dit misschien in een aparte method?

Reply:
De benodigde logic bleek al in `copyPartialStream` te zitten dus alles versimpeld. In `copyPartialStream` nog een beetje geoptimaliseerd.